source/fuzz/repeated_pass_recommender_standard.cpp

```diff
@@ -0,0 +1,326 @@
+    //   fruitful; it will be inlined in a different form
+    return RandomOrderAndNonNull({pass_instances_->GetAddFunctionCalls(),
+                                  pass_instances_->GetInlineFunctions()});
+  }
+  if (&pass == pass_instances_->GetPermuteBlocks()) {
+    // No obvious follow-on passes
+    return {};
+  }
+  if (&pass == pass_instances_->GetPermuteFunctionParameters()) {
+    // No obvious follow-on passes
+    return {};
+  }
+  if (&pass == pass_instances_->GetPermuteInstructions()) {
+    // No obvious follow-on passes
+    return {};
+  }
+  if (&pass == pass_instances_->GetPropagateInstructionsUp()) {
+    // No obvious follow-on passes
+    return {};
+  }
+  if (&pass == pass_instances_->GetPushIdsThroughVariables()) {
+    // - This pass creates synonyms, so it is worth applying them
+    return RandomOrderAndNonNull({pass_instances_->GetApplyIdSynonyms()});
+  }
+  if (&pass == pass_instances_->GetReplaceAddsSubsMulsWithCarryingExtended()) {
+    // No obvious follow-on passes
+    return {};
+  }
+  if (&pass == pass_instances_->GetReplaceCopyMemoriesWithLoadsStores()) {
+    // No obvious follow-on passes
+    return {};
+  }
+  if (&pass == pass_instances_->GetReplaceCopyObjectsWithStoresLoads()) {
+    // - We may end up with load/store pairs that could be used to create memory
+    //   copies
+    return RandomOrderAndNonNull(
+        {pass_instances_->GetReplaceLoadsStoresWithCopyMemories()});
+  }
+  if (&pass == pass_instances_->GetReplaceIrrelevantIds()) {
+    // No obvious follow-on passes
+    return {};
+  }
+  if (&pass == pass_instances_->GetReplaceLinearAlgebraInstructions()) {
+    // No obvious follow-on passes
+    return {};
+  }
+  if (&pass == pass_instances_->GetReplaceLoadsStoresWithCopyMemories()) {
+    // No obvious follow-on passes
+    return {};
+  }
+  if (&pass == pass_instances_->GetReplaceOpPhiIdsFromDeadPredecessors()) {
+    // No obvious follow-on passes
+    return {};
+  }
+  if (&pass == pass_instances_->GetReplaceOpSelectsWithConditionalBranches()) {
+    // No obvious follow-on passes
+    return {};
+  }
+  if (&pass == pass_instances_->GetReplaceParameterWithGlobal()) {
+    // No obvious follow-on passes
+    return {};
+  }
+  if (&pass == pass_instances_->GetReplaceParamsWithStruct()) {
+    // No obvious follow-on passes
+    return {};
+  }
+  if (&pass == pass_instances_->GetSplitBlocks()) {
+    // - More blocks means more chances for adding dead breaks/continues, and
+    //   for adding dead blocks
+    return RandomOrderAndNonNull({pass_instances_->GetAddDeadBreaks(),
+                                  pass_instances_->GetAddDeadContinues(),
+                                  pass_instances_->GetAddDeadBlocks()});
+  }
+  if (&pass == pass_instances_->GetSwapBranchConditionalOperands()) {
+    // No obvious follow-on passes
+    return {};
+  }
+  assert(false && "Unreachable: every fuzzer pass should be dealt with.");
```

Comment:
Add a `return` here to suppress errors in release mode.

Reply:
Done.